Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/remote/git_hub.py

```diff
@@ -289,9 +290,69 @@ def _parents_of_request(self, url, ref):
             ), url=url
         )
 
-    def request(self, method, url, data=None, params=None, auth=None, json=None, **kwargs):
-        from datetime import datetime, timedelta
+    # FIXME: Not a very flexible mock of GitHub's GraphQL API, only supports pull-request querying
+    def graphql(self, url, auth=None, json=None):
+        query = (json or {}).get('query')
+        if not query:
+            return mocks.Response.create404(url)
 
+        qline = query.splitlines()[1]
```

Comment:
This is a unit test, so we're only making an effort to make sense of a very specific graphQL query. I'm desperately trying to avoid implementing an actual graphQL parser. All we really need in this case is something that can respond to the production query we're actually sending GitHub.

Reply:
Fair, I am just worried about future contributors coming across this method and having difficulties capturing and parsing their request properly, maybe leave a TODO for future contributors?

Reply:
There is the `FIXME` right above the function declaration.